docs/courses/using_modeldb_and_modelview.rst

```diff
@@ -0,0 +1,254 @@
+
+Expand the outline by clicking on "1 cell with morphology". Then peer inside the model by clicking on each of its expandable subitems.
+
+Notice what happens when you get down to
+
+1 cell with :menuselection:`morphology --> root soma --> 6 inserted mechanisms --> hh --> gnabar`
+
+Mouse over the two graphs to explore the relationships between position and conductances gnabar. Do the same for gkbar and gl.
+
+A shortcut for discovering the distributions of spatially inhomogeneous parameters:
+
+:menuselection:`Density Mechanisms --> Heterogeneous parameters`
+
+reveals all that are spatially nonuniform. Expand any one to see the values over the model.
+
+Repeat the same exercise using NEURON's ModelView tool (:menuselection:`Tools --> Miscellaneous --> Model View`). How are the views similar? How are they different?
+
+Analyzing the Underlying Code
+
+Was this model specified by user-written hoc code, or was a CellBuilder used?
+
+Exit the simulation and search the hoc files for create statements.
+
+In the terminal execute
+
+.. code::
+    python
+
+    grep create *hoc
+
+(MSWin users: first open a bash shell, then cd to the exercises/modeldb_and_modelview/moore83 directory)
+
+Alternatively you could try Windows Explorer's semi-useful Search function, or open each hoc file with a text editor and search for create.
+
+If no hoc file contains the create keyword, maybe the CellBuilder was used.
+
+Run mosinit.hoc again and look for a CellBuilder.
+
+If you don't see one, maybe a Window Group Manager is hiding it.
+
+Click on NEURON Main Menu/Window and look for one or more window names that are missing a red check mark. If you see one, scroll down to it and release the mouse button.
+
+If a CellBuilder pops up, examine its Topology, Subsets, Geometry, and Biophysics pages.
+
+Do they agree with the output of for sec in h.allsec(): pprint(sec.psection(()) and/or what you discovered with the Model View tool?
+
+"Extra Credit" Question
+
+Now you know what's in the model cell, and how it was implemented. Suppose you wanted to get a copy of it that you could use in a program of your own. Would you do this by saving a CellBuilder to a new session file, or by using a text editor to copy create, connect, insert etc. statements from one of the hoc files?
+
+
+3. What is the user interface, how was it implemented, and how do you use it?
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+What is that panel with all the buttons?
+
+What happens if you click on one of them?
+
+Click on a different one and see what happens to the string at the top of the panel.
+
+Click on some more and see what happens to the blue dot in the PointProcessManager's shape plot.
+
+Is this one of the standard GUI tools you can bring up with the NEURON Main Menu?
+
+How does it work?
+
+    Hints: look for an xpanel statement in one of the hoc files.
+
+    Read about xpanel, xbutton, and xvarlabel in the help files.
```

Comment:
You can mark these three things with `:func:` to make them link into the relevant spots in the documentation